### PR TITLE
New bootsplash logo

### PR DESCRIPTION
**Pull request by ghost**
path212
Now it's not ugly!

### CODE REVIEW

**Comment by PJB3005** [comment]
Uh, why is your commit email wrong? 
afbeelding

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,6 @@
 [submodule "SS14.Shared.Bsdiff"]
 	path = SS14.Shared.Bsdiff
 	url = https://github.com/space-wizards/ss14.shared.bsdiff.git
+[submodule "SFML"]
```
Comment (PJB3005) [inline .gitmodules:4]: Pls no.

**Comment by ZoldorfTheWizard** [comment]
but the duct tape :(

File: .gitmodules (hunk 1 above)
Comment (Cyberboss) [reply .gitmodules:4]: \>:]

Comment (ghost) [reply .gitmodules:4]: uhhh why is this happening

**Comment by ghost** [comment]
wow why I got this weird avatar 
and why sfml wants to live so much

**Comment by clusterfack** [comment]
See #519

**Pull request closed without merging**